setup.py

```diff
@@ -307,14 +310,11 @@ def finalize_options(self):
     ),
 ]
 if not platform.system().endswith("BSD"):
-
-    RULES_PATH = LIB
-    if os.path.isfile("/etc/redhat-release"):
-        RULES_PATH = "/usr/lib"
+    RULES_PATH = pkg_config_read("udev", "udevdir")
```

Comment:
Shouldn't this be `"/" + pkg_config_read("udev", "udevdir")` since `pkg_config_read` strips off the leading `/` ?

Reply:
Not necessarily, you're right that there's an issue here, but it shouldn't be `"/" + ...` unconditionally. I need to check if we're in a virtualenv first, like in the function on line 276. I'll update the pull request shortly.